packages/babel-register/src/node.js

```diff
@@ -77,7 +79,12 @@ function compile(code, filename) {
     result.mtime = mtime(filename);
   }
 
-  maps[filename] = result.map;
+  if (result.map) {
+    if (Object.keys(maps).length === 0) {
```

Comment:
@loganfsmyth I added this change to run the install function only once.

Reply:
It's technically possible, but at the end of the day that's not what it means in our options processing. `undefined` means non-existant, so for instance if you have `sourceMaps: true` in your `.babelrc`, passing `sourceMaps: false` in the programmatic options would override `true` to `false`, whereas `sourceMaps: undefined` would have no effect.

Reply:
Is it not required to run `sourceMapSupport.install()` for each file?

Reply:
Nope, it's installed once globally, so now we install it the first time we see that we have a sourcemap.